todo.js

```diff
@@ -0,0 +1,201 @@
+const container = document.querySelector('.container');
+
+const header = document.createElement('div');
+header.className = 'header';
+container.appendChild(header);
+
+const title = document.createElement('h1');
+title.innerText = "Today's To DO";
+header.appendChild(title);
+
+const clearBtn = document.createElement('div');
+clearBtn.className = 'clear';
+header.appendChild(clearBtn);
+
+const refresh = document.createElement('i');
+refresh.className = 'fa fa-refresh';
+clearBtn.appendChild(refresh);
+
+const content = document.createElement('div');
+content.className = 'content';
+container.appendChild(content);
+
+const form = document.createElement('form');
+form.className = 'add-item';
+form.id = 'form';
+content.appendChild(form);
+
+const input = document.createElement('input');
+input.id = 'input';
+input.type = 'text';
+input.placeholder = 'Add to your list...';
+form.appendChild(input);
+
+const button = document.createElement('button');
+button.id = 'addBtn';
+button.type = 'button';
+form.appendChild(button);
+
+const enterBtn = document.createElement('i');
+enterBtn.className = 'fa-solid fa-arrow-left';
+button.appendChild(enterBtn);
+
+const list = document.createElement('div');
+list.id = 'list';
+content.appendChild(list);
+
+const clearButton = document.createElement('button');
+clearButton.id = 'clearButton';
+clearButton.type = 'button';
+clearButton.innerText = 'Clear all completed';
+container.appendChild(clearButton);
+
+export default class TodoList {
+  LIST = [];
+
+  saveToDo() {
+    const todo = JSON.stringify(this.LIST);
+    localStorage.setItem('todos', todo);
+  }
+
+  addTodo(description) {
+    const todo = {
+      description,
+      completed: false,
+      index: this.LIST.length + 1,
+    };
+    this.LIST.push(todo);
+    this.saveToDo();
+  }
+
+  getStoredTodos() {
+    this.LIST = JSON.parse(
+      localStorage.getItem('todos'),
+    );
+  }
+
+  editItem(text, index) {
+    this.LIST[index - 1].description = text.textContent;
+    this.saveToDo();
+  }
+
+  completedTodo(status, index) {
+    this.LIST[index - 1].completed = status;
+    this.saveToDo();
+  }
+
+  removeTodo = (index) => {
+    const updatedArray = this.LIST.filter((taskIndex) => taskIndex !== index);
+    updatedArray.forEach((item, index) => {
+      item.index = index + 1;
+    });
+    localStorage.setItem('todos', JSON.stringify(updatedArray));
+  };
```

Comment:
- [ ] Again, in your case, a particular task can only be deleted when completed is set to true ie when the box is checked but I think we should be able to delete a task even if it's not completed. The `Clear all completed` button at the end of your todo content should be a better place to implement this.

Reply:
All good now 👍🏽